src/components/Book/Book.js

```diff
@@ -1,22 +1,33 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-
+import { connect } from 'react-redux';
```

Comment:
- [ ] Book component should still remain presentational component. It should not connect to the store. As the prevoius comment, deleteBook should be passed down from `BookList` component as it is a container component (connects to redux store).
Please take a look at this [article](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0)